src/subspace_store.README

```diff
@@ -0,0 +1,46 @@
+A subspace store allows you to save data associated with a
+multidimensional-subspace. Subspaces are defined conceptually via a Hypercube
+(that is a collection of slices -- one for each dimension). Thus, a subspace is
+a "rectangular" cutout in a multidimensional space.
+
+that is given a hypertable with (ts Timestamp, i int) with intervals of one hour
+and 2 hash-partitions the subspaces could be:
+
+     00:00   01:00   02:00   03:00
+---|-------|-------|-------|-------|--- - -
+ 1 |       |       |       |       |
+---|-------|-------|-------|-------|--- - -
+ 2 |       |       |       |       |
+---|-------|-------|-------|-------|--- - -
+ 3 |       |       |       |       |
+---|-------|-------|-------|-------|--- - -
+
+Each subspace is cached in a tree structure, with each level of the tree
+corresponding to a dimension the hypertable is partitioned on, with the first
+level always the time dimension. Thus, the aforementioned hypertable will have
+the following tree:
+
+SubspaceStore
+    |
+    V
+SubspaceStoreInternalNode (time)
+       | (.vector)
+       V
+  |  o  | ... | ... | ... |
+     |
+     V
+  DimensionSlice (00:00 - 01:00)
+     |
+     V
+    SubspaceStoreInternalNode (dim 1)
+     |
+     V
+     .
+     .
+     .
+     |
+     V
+    ChunkInsertState (or other leaf object)
+
+the SubspaceStore has a field `descendants` storing a count of the number of
```

Comment:
"descendEnt"

Reply:
[Apparently they're both valid spellings](https://en.wiktionary.org/wiki/descendant). (Besides, that _is_ the field's name)